### DISCUSSION TITLE
Swagger / Mirth Connect / Cors

### DISCUSSION THREAD

**Discussion by 5011-5011** [nextgenhealthcare/connect#5879] [Q&A] [answered]
Hello! 
Has anyone experienced the following?
I made my OPEN API specification with Swagger.
However when I run tests from Swagger I get the following error: 

![image](https://github.com/nextgenhealthcare/connect/assets/98954172/325ec380-9715-4131-aa7c-ccc7c2dfe7a6)

I would appreciate help :)

**Comment by pacmano1** [comment, marked as answer]
The swagger web page should be presenting https, not http in that part you have crossed out.

And if you have a private IP in the URL, it really makes little sense to osbcure it, for that matter it's true on the rest of the URL.  

![image](https://github.com/nextgenhealthcare/connect/assets/44065187/65bff9af-e2e4-43a5-9ced-dba96890f99e)

**Reply by 5011-5011** [reply]
I have swagger hosted locally on the same server where I have Mirth Connect.
I did the modification as you suggested however the error persists.

![image](https://github.com/nextgenhealthcare/connect/assets/98954172/55371f13-ec28-41be-a669-7d06c6bd7431)

**Reply by pacmano1** [reply]
I don't know what you mean by "hosted locally".  The swagger stuff is hosted on the mirth server.   

I assume you opened the swagger site from the mirth admin tool menu item on the lower left called "View Client API"

**Reply by 5011-5011** [reply]
I am not referring to the Mirth Connect specification.
I made a separate specification with Swagger that describes the endpoints of a set of channels, and the specification I made and mirth connect are both hosted on the same server.

**Reply by pacmano1** [reply]
Ah, Ok.  I defer to others then :).  It is sort of a non-mirth qestion.
